frontend/src/bundles/preview/preview.tsx

```diff
@@ -61,6 +62,15 @@ const dropdownOptions = [
     },
 ];
 
+const ColumnChartDate: [string, number][] = [
+    ['Test1', 1],
+    ['Test2', 2],
+    ['Test3', 3],
+    ['Test4', 4],
+    ['Test5', 5],
+    ['Test6', 6],
+];
+
```

Comment:
It would be better to use something more meaningful for mock data, then Test1, Test2, TestN.

Reply:
Done